open_schools_platform/student_management/students/selectors.py

```diff
@@ -51,3 +52,15 @@ def get_students(*, filters=None) -> QuerySet:
     students = StudentFilter(filters, qs).qs
 
     return students
+
+
+def get_student_profiles_by_families(families: QuerySet) -> QuerySet:
+    if len(families) == 0:
+        return families
+    student_profiles_ids = \
+        filter_list_from_empty_strings(
+            list(map(lambda x: form_ids_string_from_queryset(x.student_profiles.all()), list(families)))
+        )
+    if len(student_profiles_ids) == 0:
+        raise NotFound("There are no student_profiles in provided families")
```

Comment:
Can we use NotFound like this with new exceptions?

Reply:
Yes, we do